Labfiles/07-exercise-add-to-index-use-push-api lab-files/OptimizeDataIndexing/obj/Debug/net7.0/OptimizeDataIndexing.AssemblyInfo.cs

```diff
@@ -13,7 +13,7 @@
 [assembly: System.Reflection.AssemblyCompanyAttribute("OptimizeDataIndexing")]
 [assembly: System.Reflection.AssemblyConfigurationAttribute("Debug")]
 [assembly: System.Reflection.AssemblyFileVersionAttribute("1.0.0.0")]
-[assembly: System.Reflection.AssemblyInformationalVersionAttribute("1.0.0")]
+[assembly: System.Reflection.AssemblyInformationalVersionAttribute("1.0.0+8e767caa68d14a7dcbebb19c12606f640a6efe3d")]
```

Comment:
is this going to screw any learners up? I'm not familiar with this notation

Reply:
debug files - now deleted